### PR TITLE
Update teste.c

### PR DESCRIPTION
**Pull request by sabino**

### CODE REVIEW

**Comment by sabino** [comment]
could you review @mdib?

**Review by MDIB — CHANGES_REQUESTED** [review]
:no:

**Review by MDIB — CHANGES_REQUESTED** [review]
req

File: teste.c [hunk 1]
```diff
@@ -2,3 +2,6 @@ fixes bug #1
```
Comment (sabino) [inline teste.c]: Testing comment

File: teste.c [hunk 2]
```diff
@@ -2,3 +2,6 @@ fixes bug #1
 
 teste de commit
 ainda testando
+
+
+testing
```
Comment (sabino) [inline teste.c:7]: Testing comment

Comment (sabino) [inline teste.c:7]: Testing comment

**Pull request closed without merging**